CHANGELOG.rst

```diff
@@ -7,6 +7,9 @@ Changelog
 .. note:: This version is not yet released and is under active development.
 
 * Added support for Python 3.6.
+* Changed ASN.1 dependency from ``pyasn1`` to ``asn1crypto`` resulting in a
+  general performance increase when encoding/decoding ASN.1 structures. Also,
+  the pyasn1_modules test dependency is no longer required.
```

Comment:
I think it doesn't like the spelling of `pyasn1_modules`, can you put it in backticks as well?